examples/yacat/yacat.py

```diff
@@ -67,8 +67,9 @@ async def worker_find_password(ctx: WorkContext, tasks):
 
             # Commands to be run on the provider
             commands = (
+                "rm /golem/work/hashcat.potfile || true; "
                 "touch /golem/work/hashcat.potfile; "
-                f"hashcat -a 3 -m 400 /golem/work/in.hash --skip {skip} --limit {limit} {args.mask} -o /golem/work/hashcat.potfile"
+                f"hashcat -a 3 -m 400 /golem/work/in.hash {args.mask} --skip={skip} --limit={limit} --self-test-disable -o /golem/work/hashcat.potfile || true"
```

Comment:
Are those `=`'s really needed? I believe they do no harm, but do they actually solve any issue?

Reply:
After testing I can say it's not. To be in the same page, aligned them as in the description of hashcat. long flags are represented with equal, short equivalents are without. But if unnecessary we can revert it.